README.md

```diff
@@ -16,9 +16,17 @@ composer require --dev 10up/wp_mock
 
 Learn more about how to configure and how to use WP_Mock by reading [the WP_Mock documentation](https://wp-mock.gitbook.io/documentation/getting-started/introduction).
 
+## Support Level
+
+**Active:** 10up and GoDaddy are actively working on this, and we expect to continue work for the foreseeable future including keeping tested up to the most recent version of WordPress.  Bug reports, feature requests, questions, and pull requests are welcome.
+
+## Changelog
+
+A complete listing of all notable changes to Distributor are documented in [CHANGELOG.md](https://github.com/10up/wp_mock/blob/trunk/CHANGELOG.md).
+
```

Comment:
I was planning on using the automatic changelogs generated by GitHub for each release, as in https://github.com/10up/wp_mock/releases/tag/0.5.0 - this avoids the manual chore of having to maintain an identical file that is just a collection of the same things - in the end it's not a WordPress plugin where it may be helpful to have a list of changes outside of the repo

happy to re-include, but this (re)introduces a small burden because each change needs a follow up PR just to update the file, re-approve and merge which could be avoided IMO

Reply:
Those are areas we spell out as expected in https://10up.github.io/Open-Source-Best-Practices/community/#readme, https://10up.github.io/Open-Source-Best-Practices/releasing/#changelog, and https://10up.github.io/Open-Source-Best-Practices/github-process/#support-levels.  So perhaps we remove/reduce those initial references?

Reply:
never mind I think since the project is hosted by 10up makes sense for consistency to include these then

for the CHANGELOG.md in the end we won't have many releases and I figure it's ok to maintain